run_analysis.py

```diff
@@ -154,6 +142,22 @@ def analysis(config): #{{{
     config = MpasAnalysisConfigParser()
     config.read(configFileNames)
 
+    # choose the right rendering backend, depending on whether we're displaying
+    # to the screen
+    if not config.getboolean('plot', 'displayToScreen'):
+        mpl.use('Agg')
+    import matplotlib.pyplot as plt
+    # analysis can only be imported after the right MPL renderer is selected
+    from mpas_analysis.ocean.ohc_timeseries import ohc_timeseries
+    from mpas_analysis.ocean.sst_timeseries import sst_timeseries
+    # from mpas_analysis.ocean.nino34_timeseries import nino34_timeseries
+    # from mpas_analysis.ocean.mht_timeseries import mht_timeseries
+    # from mpas_analysis.ocean.moc_timeseries import moc_timeseries
+    from mpas_analysis.ocean.ocean_modelvsobs import ocn_modelvsobs
+
+    from mpas_analysis.sea_ice.timeseries import seaice_timeseries
+    from mpas_analysis.sea_ice.modelvsobs import seaice_modelvsobs
+
```

Comment:
@xylar, I agree with your concern about not having imports at the top of the file and agree it is aesthetically unpleasing.  However, I also agree with you that there isn't a clear way to do this better because all our analysis depends on matplotlib even if tangentially.  A bigger concern is probably the commented out lines and lack of organization.  

One way to help with these problems that may be a good compromise would be to have a function right after the import section at the top of the file

```python
def load_analysis_modules():
     import matplotlib.pyplot as plt
     # analysis can only be imported after the right MPL renderer is selected
     from mpas_analysis.ocean.ohc_timeseries import ohc_timeseries
     from mpas_analysis.ocean.sst_timeseries import sst_timeseries
     # from mpas_analysis.ocean.nino34_timeseries import nino34_timeseries
     # from mpas_analysis.ocean.mht_timeseries import mht_timeseries
    # from mpas_analysis.ocean.moc_timeseries import moc_timeseries
     from mpas_analysis.ocean.ocean_modelvsobs import ocn_modelvsobs
 
     from mpas_analysis.sea_ice.timeseries import seaice_timeseries
     from mpas_analysis.sea_ice.modelvsobs import seaice_modelvsobs
```

`load_analysis_modules()` could be called essentially at line 149 after the selection of the backend.  This function would be the first function in the file and may not even have a whitespace after the import section.  I'm not sure what makes the most sense but probably would leave one space, not the standard two to designate it is associated with the imports section.

What do you think of this idea?  I think it may be the compromise that cleans up this code and helps clarify what is happening by putting each piece closer to its logical and standard position.